save-backend/src/main/kotlin/org/cqfn/save/backend/repository/TimestampBasedFileSystemRepository.kt

```diff
@@ -54,6 +67,13 @@ class TimestampBasedFileSystemRepository(configProperties: ConfigProperties) {
         .filter { it.isDirectory() }
         .flatMap { it.listDirectoryEntries() }
 
+    /**
+     * @param owner owner name
+     * @return image of avatar for owner in [rootDirImage]
+     */
+    fun getAvatar(owner: String): ImageInfo? = organizationRepository.findByName(owner)
```

Comment:
How is this connected with `TimestampBasedFileSystemRepository`? This method should be in organizationService